QUALITY_GUIDELINES.md

```diff
@@ -0,0 +1,21 @@
+This page lists criteria that code, commits, and pull requests should fulfill.
```

Comment:
Not sure I adhere to the standard below. But I like them.

Reply:
We do the best we can ;)